src/main/java/Boop.java

```diff
@@ -0,0 +1,106 @@
+            String[] comArr = command.split(" ");
+            String curCommand = comArr[0];
+            if(curCommand.equals("bye")) {
+                break;
+            } else if (curCommand.equals("list")) {
+                printLst();
+            } else if (curCommand.equals("mark")) {
+                mark(comArr);
+            } else if (curCommand.equals("unmark")) {
+                unmark(comArr);
+            } else {
+                addTask(curCommand, command);
+            }
+            System.out.println("____________________________________________________________");
+        }
+        System.out.println("Bye. Hope to see you again soon!");
+        System.out.println("____________________________________________________________");
+    }
+
+    public static void printLst() {
+        System.out.println("Here are the tasks in your list:");
+        for(int i = 0; i < count; i += 1) {
+            Task t = tasks[i];
+            System.out.println("" + (i + 1) + ". " + t.toString());;
+        }
+    }
+
+    public static void mark(String[] arr) {
+        if(arr.length <= 1) {
+            System.out.println("Please specify which item to mark");
+            return;
+        }
+        int markNum;
+        try {
+            markNum = Integer.parseInt(arr[1]);
+        } catch (NumberFormatException e) {
+            System.out.println("Please provide a number to mark");
+            return;
+        }
+        if (markNum >= 1 && markNum <= count) {
+            tasks[markNum - 1].markTask();
+        } else {
+            System.out.println("This number is not on the list");
+        }
+    }
+
+    public static void unmark(String[] arr) {
+        if(arr.length <= 1) {
+            System.out.println("Please specify which item to unmark");
+            return;
+        }
+        int markNum;
+        try {
+            markNum = Integer.parseInt(arr[1]);
+        } catch (NumberFormatException e) {
+            System.out.println("Please provide a number to unmark");
+            return;
+        }
+        if (markNum >= 1 && markNum <= count) {
+            tasks[markNum - 1].unmarkTask();
+        } else {
+            System.out.println("This number is not on the list");
+        }
+    }
+
+    public static void addTask(String com, String fullInput) {
+        Task t;
+        if (com.equals("todo")) {
+            String desc = fullInput.substring(5);
+            t = new Todo(desc);
+        }else if (com.equals("deadline")) {
+            String desc = fullInput.substring(9);
+            String[] descArr = desc.split(" /by ");
+            t = new Deadline(descArr[0], descArr[1]);
+        } else if (com.equals("event")) {
+            String desc = fullInput.substring(6);
+            String[] descArr = desc.split(" /");
+            String from = descArr[1].substring(5);
+            String to = descArr[2].substring(3);
+            t = new Event(descArr[0], from, to);
```

Comment:
Try to avoid using single letter variable names, make the names more meaningful.